recommendation/mozlog/middleware.py

```diff
@@ -7,6 +7,13 @@
 
 IS_PROTOCOL = r'^[^\s]+\:\S'
 IS_HOSTNAME = r'^[^\s]+\.\S'
+LOG_PATH_BLACKLIST = [
+    '/favicon.ico',
+    '/__heartbeat__',
+    '/__lbheartbeat__',
+    '/nginx_status',
+    '/robots.txt'
```

Comment:
As far as I can see, nothing automated hits it so it doesn't really matter.  Once we launch I expect we might have another URL or two to add (I had forgotten about favicon and robots.txt until I looked)